parsing/ast_helper.mli

```diff
@@ -322,7 +322,7 @@ module Mb:
 module Opn:
   sig
     val mk: ?loc: loc -> ?attrs:attrs -> ?docs:docs ->
-      ?override:override_flag -> lid -> open_description
+      ?override:override_flag -> module_expr -> open_description
```

Comment:
Similarly, I think it would be useful to have specialized function for the `lid` case. Moreover, since `mk` is exposed in compiler-libs, I am wondering if it would not be more pratical for users to add a `mk_expr` for the generic case and let `mk` as the specialized version.